src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepository.scala

```diff
@@ -41,6 +41,21 @@ class FileMetadataRepository(db: Database)(implicit val executionContext: Execut
     db.run(query.result)
   }
 
+  def getClosureMetadataProperty: Future[Seq[FilepropertyRow]] = {
```

Comment:
I would be tempted to move these queries out into a new repo class: `CustomMetadataPropertiesRepository`. This will make it clearer what these are.